lib/LTP/utils.pm

```diff
@@ -121,6 +122,8 @@ sub log_versions {
     record_info('KERNEL DEFAULT PKG', script_output("cat $kernel_pkg_log", proceed_on_failure => 1));
     record_info('KERNEL EXTRA PKG', script_output('rpm -qi kernel-default-extra', proceed_on_failure => 1));
 
+    record_info('KERNEL pkg', script_output('rpm -qa | grep kernel', proceed_on_failure => 1));
```

Comment:
`KERNEL DEFAULT PKG` above will show package info for `$kernel_pkg`. The info entry just needs to be renamed.

Reply:
Wait, never mind.

Reply:
I'm sorry, I was just too lazy to put more code for proper label of `KERNEL DEFAULT PKG`. And `rpm -qa | grep kernel` might be useful for some new product.